src/styles/_catalog.scss

```diff
@@ -1,22 +1,46 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  justify-content: center;
```

Comment:
give max-width to container to your project look similar to original

Reply:
_catalog.scss isn't connected to the project and will be removed. The Requirements is: "write styles in src/styles/main.scss"